src/redux/uniqueTokens.ts

```diff
@@ -322,6 +322,7 @@ export const fetchUniqueTokens = (showcaseAddress?: string) => async (
   if (currentNetwork === Network.mainnet) {
     const poaps = (await fetchPoaps(accountAddress)) ?? [];
     if (poaps.length > 0) {
+      analytics.identify(undefined, { poaps: uniqueTokens.length });
```

Comment:
We should use `analytics.track(eventName, { poaps })` here. That should be easier to parse out later on. Plus, we're otherwise adding unnecessary monthly tracked users (MTUs) which costs $.

Reply:
Daniel messaged and clarified — I might be incorrect here.

Do we know how Segment handles anonymous IDs in RN? If we identify each time we fetch here, are we getting a new identity or is it reusing the old? I guess I thought our events weren't fixed to any identifier, but maybe that's wrong 🤷

Reply:
I see what you're saying im p sure that undefined in this instance will reuse the `analyticsUserIdentifier` we set previously. I'll refactor so thats 100% the case. imo this should be in identity vs an event though (whole things a mess when we consider multiple user accounts, etc)

another thing we try to emphasis is keeping user data at a minimum so I think we'd prefer to only send the length than the full object of assets